### DISCUSSION TITLE
请问为什么文件上传文件后一直在索引

### DISCUSSION THREAD

**Discussion by XiaoLiuH** [GaiZhenbiao/ChuanhuChatGPT#370] [General]
![image](https://user-images.githubusercontent.com/108645872/227543005-597caa18-b4c1-4faf-91bc-e0c21d76a6f6.png)

**Comment by GaiZhenbiao** [comment]
不应该呀，构建过一次之后就不会重复构建了，而是从缓存中加载。建议刷新下网页，大概率是网络问题。另外你的问法不太对，应该问“这篇文章都讲了什么”这样的问题，GPT并不知道文件名。

**Reply by XiaoLiuH** [reply]
好呢，谢谢您
